packages/dds/sequence/src/generateSequentialId.ts

```diff
@@ -0,0 +1,144 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
there is very similar code already existing in merge tree for ordinal generation, i wonder if that could be encapsulated and reused

Reply:
[https://github.com/microsoft/FluidFramework/blob/e5a7147aad10d055a12086d37e64080317131bb9/packages/dds/merge-tree/src/mergeTree.ts#L347](https://github.com/microsoft/FluidFramework/blob/e5a7147aad10d055a12086d37e64080317131bb9/packages/dds/merge-tree/src/mergeTree.ts#L347)

---
In reply to: [546008298](https://github.com/microsoft/FluidFramework/pull/4641#discussion_r546008298) [](ancestors = 546008298)

Reply:
I looked at the ordinal code. It was similar but very inside mergeTree and not really re-usable now so will leave this as a follow up